storybook/stories/Table.stories.js

```diff
@@ -0,0 +1,108 @@
+import { storiesOf } from '@storybook/vue'
+import Table from '@mesg-components/table/Table'
```

Comment:
This library is not included in the package.json

Reply:
Done.